airflow/gcp/hooks/bigtable.py

```diff
@@ -276,3 +278,58 @@ def get_cluster_states_for_table(instance: Instance, table_id: str) -> Dict[str,
 
         table = Table(table_id, instance)
         return table.get_cluster_states()
+
+    def create_rowkey_regex_filter(self, regex):
```

Comment:
Where is the helper used?

Reply:
This helper (and the new one like it) is only used outside the Hook, for creating RowFilter objects that can be passed in to the `check_and_mutate_row` method.